### PR TITLE
[jsk_perception] Publish the difference between start and current image

### PR DESCRIPTION
**Pull request by wkentaro**
upper_left: raw_image
upper_center: mask_image
upper_right: mask_applied_image
lower_right: hue of mask_applied_image
lower_center: stored_image
lower_left: diff of stored_image

### CODE REVIEW

**Comment by wkentaro** [comment]
position1

**Comment by wkentaro** [comment]
screenshot from 2015-05-08 02 53 15

File: jsk_perception/scripts/image_time_diff.py [hunk 1]
```diff
@@ -0,0 +1,77 @@
...
+        pub_stored = self.pub_stored
+        for ns, (pub_diff, img_stored, pub_debug) in pub_stored.items():
+            # compute diff for all publications
+            diff = cv2.absdiff(img, img_stored)
+            pub_diff.publish(bridge.cv2_to_imgmsg(diff))
+            pub_debug.publish(bridge.cv2_to_imgmsg(img_stored))
+
+    def spin(self):
+        rate = rospy.Rate(rospy.get_param('rate', 10))
+        while not rospy.is_shutdown():
```
Comment (garaemon) [inline jsk_perception/scripts/image_time_diff.py]: Use rospy.spin()

File: jsk_perception/scripts/image_time_diff.py [hunk 2]
```diff
@@ -0,0 +1,77 @@
...
+            self.spin_once()
```
Comment (wkentaro) [inline jsk_perception/scripts/image_time_diff.py]: But I need to process image files here.
How can I use rospy.spin() when I need process something?

Comment (garaemon) [inline jsk_perception/scripts/image_time_diff.py]: I see, another option is to use timer, but it's ok

**Pull request merged**